deltachat-ios/Controller/ContactDetailViewController.swift

```diff
@@ -608,4 +610,30 @@ extension ContactDetailViewController: MultilineLabelCellDelegate {
             UIApplication.shared.open(url)
         }
     }
+
+    private func selectProxy(url proxyURL: String) {
```

Comment:
let's unduplicate that code, eg. move it beside handleDeltaChatInvitation().

this makes maintenance easier, esp. as the code from the profile is probably used far less often (still, i think, it is nice to allow that also there)

Reply:
I thought about that, my rule of thumb is: If it appears more than twice, it's worth it to unduplicate it. In this case we'd probably need to unduplicate the entire `urlTapped`-thing, right?